### PR TITLE
As a user I want to filter content by Planning Agenda. [SDESK-5055]

### PR DESCRIPTION
**Pull request by GyanP**

### CODE REVIEW

File: apps/content_filters/filter_condition/filter_condition_parameters.py [hunk 1]
```diff
@@ -205,3 +212,11 @@ def _get_stage_field_values(self, desks):
                 continue
             stages[i]['name'] = '{}: {}'.format(desk['name'], stage['name'])
         return tuple(i for i in stages if i)
+
+    def _get_planning_agendas(self):
+        agendas = []
+        agenda_service = get_resource_service('agenda')
+        cursor = agenda_service.find({'is_enabled': True})
```
Comment (petrjasek) [inline apps/content_filters/filter_condition/filter_condition_parameters.py]: you can do this via `return list(agenda_service.find({'is_enabled': True}))`

File: apps/content_filters/filter_condition/filter_condition_parameters.py [hunk 2]
```diff
@@ -137,6 +138,12 @@ def get(self, req, lookup):
                   {'field': 'anpa_take_key',
                    'operators': ['in', 'nin', 'eq', 'ne', 'like', 'notlike', 'startswith', 'endswith']
                    },
+                  {'field': 'agendas',
```
Comment (petrjasek) [inline apps/content_filters/filter_condition/filter_condition_parameters.py]: actually you could add this field only if agenda is enabled, or even better create some api which planning could use to register a field..

Comment (petrjasek) [reply apps/content_filters/filter_condition/filter_condition_parameters.py]: but given that it's the first external field, I would probably just make it conditional and comment on adding api if needed next time..

Comment (GyanP) [reply apps/content_filters/filter_condition/filter_condition_parameters.py]: Is there a way to check if agendas or enabled or not? If not I can simply check the length of `agenda_values` that is coming from here `agenda_values = self._get_planning_agendas()`. Would it be fine?

Comment (petrjasek) [reply apps/content_filters/filter_condition/filter_condition_parameters.py]: I think it's this `if 'planning' in app.config.get('INSTALLED_APPS', []):` you do elsewhere

**Pull request merged**